eng/Versions.props

```diff
@@ -6,7 +6,10 @@
   </PropertyGroup>
   <!-- Repo Version Information -->
   <PropertyGroup>
-    <VersionPrefix>0.1.0</VersionPrefix>
+    <VersionMajor>0</VersionMajor>
```

Comment:
Is a "0" a legal major number?

Reply:
yes. for example https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase